client/src/app/project-create-edit/schemas/project-ingest-schema.json

```diff
@@ -233,9 +232,9 @@
       "example": "1"
     },
     "cellCount": {
-      "description": "Specify cell count",
+      "description": "Please use the new estimated cell count field.",
       "type": "integer",
-      "user_friendly": "Cell count",
+      "user_friendly": "Cell count (TO BE DEPRECATED)",
```

Comment:
I will remove this field once we have copied the old cell count field to the new estimated_cell_count of existing projects.